File: app/src/main/java/farasense/mobile/view/components/FaraSenseButtonBold.java
package farasense.mobile.view.components;

import android.content.Context;
import android.graphics.Typeface;
import android.support.v7.widget.AppCompatButton;
import android.util.AttributeSet;
import android.widget.Button;

public class FaraSenseButtonBold extends AppCompatButton {

    public FaraSenseButtonBold(Context context, AttributeSet attrs) {
        super(context, attrs);
        this.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/Montserrat-ExtraBold.ttf"));
    }
}

File: app/src/main/java/old_farasense/view/components/FaraSenseButtonBold.kt
package old_farasense.view.components

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatButton

class FaraSenseButtonBold(context: Context, attrs: AttributeSet) : AppCompatButton(context, attrs) {
    init {
        this.typeface = Typeface.createFromAsset(context.assets, "fonts/Montserrat-ExtraBold.ttf")
    }
}
